src/Socket_p.h

```diff
@@ -548,7 +545,7 @@ namespace Arcus
 
         google::protobuf::io::ArrayInputStream array(wire_message->data, wire_message->size);
         google::protobuf::io::CodedInputStream stream(&array);
-        stream.SetTotalBytesLimit(message_size_maximum, message_size_warning);
```

Comment:
The second parameter was deprecated in version 3.5, so this should not be a problem in our Protobuf version.